### PR TITLE
rename local variables so they do not conflict with the class ones.

### PR DESCRIPTION
**Pull request by bigfluffycookie**

### CODE REVIEW

File: Exercise/VSIX/AnalyzeCommand.cs [hunk 1]
```diff
@@ -82,22 +82,22 @@ public static async Task InitializeAsync(AsyncPackage package)
...
+            var dteService = package.GetService<DTE, DTE2>();
+            IAnalysisController analysisControllerComp = null;
+            ILogger loggerComp = null;
 
             try
             {
                 var comp = package.GetService<SComponentModel, IComponentModel>();
-                analysisController = comp.GetService<IAnalysisController>();
-                logger = comp.GetService<ILogger>();
+                analysisControllerComp = comp.GetService<IAnalysisController>();
```
Comment (duncanp-sonar) [inline Exercise/VSIX/AnalyzeCommand.cs:92]: You shouldn't need to fetch the controller here - it's passed in the constructor.

Comment (bigfluffycookie) [reply Exercise/VSIX/AnalyzeCommand.cs:92]: I do not understand, it is passed into the constructor on line 100, with this variable? If I remove it who is going to fetch it

Comment (duncanp-sonar) [reply Exercise/VSIX/AnalyzeCommand.cs:92]: Why do you need to fetch it here, when it has already been passed in the constructor?

Comment (bigfluffycookie) [reply Exercise/VSIX/AnalyzeCommand.cs:92]: But the passing it in the constructor happens after I fetch it here? Or am I completely missing something

Comment (duncanp-sonar) [reply Exercise/VSIX/AnalyzeCommand.cs:92]: Sorry - ignore me, I'm talking rubbish. I was misreading the code. Apologies for the noise.

**Pull request merged**